README.md

```diff
@@ -1,29 +1,97 @@
-# Deno
+<h1 align="center">
+  <a name="logo" href="https://deno.land"><img src="https://deno.land/logo.svg" alt="Deno Land" width="200"></a>
+  <br>
+ A secure runtime for JavaScript and TypeScript 
+</h1>
+
+<div align="center">
+  <h4>
+    <a href="https://github.com/denoland/deno/actions"><img src="https://github.com/denoland/deno/workflows/ci/badge.svg?branch=master&event=push"/></a>
+    <a href="https://github.com/denoland/deno/stargazers"><img src="https://img.shields.io/github/stars/denoland/deno.svg?style=plasticr"/></a>
+    <a href="https://github.com/denoland/deno/commits/master"><img src="https://img.shields.io/github/last-commit/CCOSTAN/Home-AssistantConfig.svg?style=plasticr"/></a>
```

Comment:
This is also not deno related

Reply:
> https://github.com/denoland/deno/blob/c7af17cc960d080a851ad1ffabd686161e2b1f6e/README.md
> 
> I'm sorry but these are bad changes. What are the table of contents and headers for? Most of the added badges provide info that is already visible on the main page.
What do you suggest ?

Reply:
Not sure what this comment has to do with this clearly unrelated URL:
`https://img.shields.io/github/last-commit/CCOSTAN/Home-AssistantConfig.svg`